autospotting.go

```diff
@@ -123,6 +125,12 @@ func (c *cfgData) parseCommandLineFlags() {
 			"\tcurrent: the same as initial on-demand instances\n"+
 			"\t<instance-type>: the actual instance type to use")
 
+	flag.StringVar(&c.DisallowedInstanceTypes, "disallowed_instance_types", "",
+		"If specified, the spot instances will _never_ be of this type. "+
+			"This should be a list of instance types (comma or whitespace separated, "+
+			"also supports globs).\n\t"+
+			"Example: ./autospotting -disallowed-instance-types 't2.*,c4.xlarge'")
```

Comment:
I think this actually needs to match entirely the one above (use underscores instead of dashes in the example)

Reply:
Ok done